spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -247,31 +244,25 @@
                                     ["Enter the date you started at this school or organisation", "#jobseekers-profile-employment-form-started-on-field-error"])
             end
           end
-
-          it "makes reason mandatory when non current" do
```

Comment:
Reason is now mandatory whether employment is current or not, so removed this test.